### ISSUE TITLE
The pagination widget for finished runs no longer shows the last page

### ISSUE THREAD

**Issue by vdbergh** [glinscott/fishtest#1679]
Note that pagination for the finished runs always worked well (in contrast to the event log).

![Screenshot from 2023-05-09 18-42-38](https://github.com/glinscott/fishtest/assets/6613103/cc117ced-3f7e-454d-9684-ea104a87c0a3)

**Comment by dav1312** [comment]
https://github.com/glinscott/fishtest/pull/1631

**Comment by vdbergh** [comment]
> #1631

Yes but this was meant to be for the event log. Not for the finished runs (for which there was never any problem).

**Comment by dav1312** [comment]
I'm just linking the PR bcs u didn't in the issue description.
Also the PR doesn't seem to indicate that it was meant only for the events log, it just says "pagination" in general.
The last page (currently 5550) of finished tests has a TTFB of 300ms and the same 5550 page in events has a TTFB of 4 seconds, maybe there are some underlying issues in the events.

**Comment by vdbergh** [comment]
Yes we tried several times to fix this with appropriate indexes, but without success.

**Comment by vondele** [comment]
finished runs had the same problem:
```
$ time wget -q -O - https://tests.stockfishchess.org/tests/user/Vizvezdenec?page=817 > /dev/null

real	0m19.507s
user	0m0.009s
sys	0m0.003s

```

**Comment by vdbergh** [comment]
Yes of course for the user pages.

But there was never a problem for the generic page.

**Comment by vdbergh** [comment]
I always found it nice that with a single click one could observe the humble beginnings of Fishtest. It is a bit like a museum.

It would be trivial to fix it using an optional argument that restores the old behavior when that is desired.

**Comment by ppigazzini** [comment]
fixed in https://github.com/official-stockfish/fishtest/pull/2453

<img width="443" height="209" alt="Image" src="https://github.com/user-attachments/assets/2188d970-14d6-4864-9f79-0c1232d6e0e2" />
